packages/e2e/cypress/integration/ui/components/authenticator/reset-password/reset-password.steps.ts

```diff
@@ -1,5 +1,14 @@
-import { Then } from 'cypress-cucumber-preprocessor/steps';
+import { Then, When } from 'cypress-cucumber-preprocessor/steps';
 
 Then('I will be redirected to the confirm forgot password page', () => {
-  cy.findByRole('textbox', { label: 'New password' }).should('exist');
+  cy.findInputField('New Password').should('exist');
+});
```

Comment:
Reset password screen has different labels from other screens, so I had to write specific tests for those. 

On 3.0 major bump, we can look to make these labels consistent.